jekyll/_cci2/pipelines.adoc

```diff
@@ -0,0 +1,146 @@
+** Authorize your connection if this is not already showing with a icon:check-circle[]
+** Select your repository
++
+CAUTION: **Select the same repository for your configuration source and trigger** (described in the following section). There is currently a bug that means that CircleCI will always look for the configuration source in the same repository from which the triggering event was emitted.
+** (Optional) If you will be assigning an inbound webhook trigger to this configuration source, select a fallback branch to use in case the triggering event does not have a triggering branch.
+** Enter the path to your configuration file relative to the root of your repository. For example, `.circleci/config.yml`. If you are integrated through the GitHub App, you can enter any path to a `.yml` file, and the file can have any name you choose.
+. Click btn:[Save]
+
+Once you have set up a configuration source you need to set up a project trigger to connect it to. This is described in the following section.
+
+=== Project triggers
+
+TIP: If you created a project xref:create-project#[in the CircleCI web app] you will find a trigger is already set up.
+
+Triggers are rules that define which configuration source should be run on a given event. A project can have one or more triggers, which are associated with events from a source of change. Triggers include, but are not limited to, a push to a branch on a version control system (VCS), for example, a GitHub repository. A trigger determines an event (or events) along with a configuration source to define a pipeline. A trigger can have only one configuration source but a configuration source can have multiple triggers.
+
+A trigger kicks off a pipeline to run the workflows defined by your connected configuration source.
+
+image::{{site.baseurl}}/assets/img/docs/project-settings-triggers.png[Triggers]
+
+==== GitHub App project triggers
+
+When the CircleCI GitHub App is installed for your organization, GitHub starts to send events for the repositories you have granted access to. CircleCI listens for push and tag events. When a trigger is created CircleCI has enough information to use the event data to determine if a pipeline should be triggered.
+
+==== GitLab project triggers
+
+When a trigger is created, CircleCI registers a webhook with GitLab. Push events from GitLab are sent to CircleCI. CircleCI then uses the event data to determine _if_ a pipeline should run, and if so, _which_ pipeline should be run.
+
+In addition to a configuration source, each trigger includes the webhook URL, and in this scenario, a CircleCI-created GitLab token. The webhook URL and GitLab token are used to securely register the webhook within GitLab in order to receive push events from your GitLab repository.
```

Comment:
This is from the original GitLab integration docs and was just provided to give wider context about how the integration works I believe. But now you mention it, it could be misleading. We could remove?

Reply:
<img width="498" alt="Screenshot 2024-05-07 at 13 18 28" src="https://github.com/circleci/circleci-docs/assets/19594309/a54f14af-4dbd-4adb-97d0-3cd745ed05bf">

Reply:
I will add a note about this modal being shown in error, and we can remove when it's removed